lecture_notes/Exercises/lesson_1_start.md

```diff
@@ -33,24 +32,25 @@ Hints:
 
 ```python
 PI = 3.14 # definování konstanty - proměnná, která se nemění
-polomer = float(input("Zadejte poloměr: "))
-obvod = 2 * PI * polomer
-obsah = PI * polomer ** 2 # ** je mocnina, polomer ** 2 znamená polomer na druhou
-objem = 4 / 3 * PI * polomer ** 3
-print("Obvod kružnice o poloměru " + str(polomer) + " je " + str(obvod))
-print("Obsah kruhu o poloměru " + str(polomer) + " je " + str(obsah))
-print("Objem koule o poloměru " + str(polomer) + " je " + str(objem))
+# 1. získat poloměr od uživatele.
+
+# 2. Vypočítat obvod kruhu
+
+# 3. Vypočítat obsah kruhu
+
+# 4. Vypočítat objem koule
+
+# 5. Vytisknutí všech hodnot.
+
 ```
 
 ## Minutes to hours and minutes
 * Write a program that receives a number representing **minutes**. The goal is to convert minutes into **hours and minutes**. You will need `input()`, `print()`, `str()` and `int()` for casting. You will further need these operators: integer division (`//`) and modulo (`%`) (modulo calculates the reminder after division, eg. `5%3 = 2` - the remainder after dividing `5` by `3` is `2`)
 
 ```python
-POCET_MINUT_V_HODINE = 60 # definování konstanty - proměnná, která se nemění
-
-minuty = int(input("Zadejte počet minut: "))
-pocet_hodin = minuty // POCET_MINUT_V_HODINE # celočíselné dělení
-pocet_minut = minuty % POCET_MINUT_V_HODINE # zbytek po dělení
+# 0. definování konstanty počtu minut v hodině - proměnná, která se nemění, není nutné, ale je to lepší.
```

Comment:
Tady bych to rovnou napsal. Řekněme, že je to ukázka best practices
`POCET_MINUT_V_HODINE = 60`
Nechybí tam náhodou taky krok s inputem?